layout/Library/Application Support/YouPiP.bundle/en.lproj/Localizable.strings

```diff
@@ -1,5 +1,7 @@
 // Settings
 
+"SETTINGS_TITLE" = "YouPiP";
```

Comment:
You'll want to add this line to every localization file, not just English and your language.

Reply:
Ofc, I will do that.
I preferred to ask first if my concept is right and will be accepted :)

Reply:
I agree. Sure.